### PR TITLE
[NIFI-6237] - styling for td-chips text input component (no text over…

### PR DESCRIPTION
**Pull request by rfellows**
…lap with filter icon, width matches underline width).

Thank you for submitting a contribution to Apache NiFi Flow Design System.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [X] Is there a JIRA ticket associated with this PR? Is it referenced
     in the commit message?

- [X] Does your PR title start with either NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [X] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [X] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] Have you ensured that a full build and that the full suite of unit tests is executed via npm run clean:install at the root nifi-fds folder?
- [ ] Have you written or updated the Apache NiFi Flow Design System demo application to demonstrate any new functionality, provide examples of usage, and to verify your changes via npm start at the nifi-fds/target folder?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file under nifi-fds?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file found under nifi-fds?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: src/platform/core/common/styles/_inputs.scss [hunk 1]
```diff
@@ -107,3 +107,12 @@ body[fds] td-chips .mat-input-placeholder-wrapper::after {
...
 }
+
+body[fds] td-chips {
+  input.mat-input-element,
+  textarea.mat-input-element {
+    /* subtract 42 (10 for padding-left, 30 for padding right, 2 the 1px border of the input on the left & right) */
+    width: calc(100% - 42px);   // account for padding and border as to line up with the underline
+    padding-right: 30px;        // Don't allow the text to overlap the filter icon
+  }
+}
```
Comment (scottyaslan) [inline src/platform/core/common/styles/_inputs.scss]: I think you will want to add an @mixin so you can:
```
input.mat-input-element:focus,
    textarea.mat-input-element:focus {
      border-color: $accentColor;
    }
```

**Comment by scottyaslan** [comment]
https://github.com/apache/nifi-fds/blob/ab24910f19cf38d07d3fa5c66f6a2b4cf3e7b32e/src/platform/core/common/styles/_inputs.scss#L69

I think it should be 29px

**Comment by scottyaslan** [comment]
https://github.com/apache/nifi-fds/blob/ab24910f19cf38d07d3fa5c66f6a2b4cf3e7b32e/src/platform/core/common/styles/_inputs.scss#L103

I think it should be: 13px 10px 0px 0px

**Comment by scottyaslan** [comment]
Also, please add:
```
.td-chip span {
      font-size: 10px;
    }
```
to _inputs.scss

**Comment by scottyaslan** [comment]
And please add:
```
mat-form-field.td-chips-form-field.mat-input-container {
      width: unset;
    }
```

**Comment by scottyaslan** [comment]
I think we should move the td-chips stuff out into its own file and @mixin... _td-chips.scss ... what do you think?

File: src/platform/core/common/styles/_inputs.scss (hunk 1 above)
Comment (rfellows) [reply src/platform/core/common/styles/_inputs.scss]: I actually did that in another PR (https://github.com/apache/nifi-fds/pull/16).

**Comment by rfellows** [comment]
I think adding a _td-chips.scss makes sense. I can move the td-chips related stuff into there. The suggestions for the non td-chips I could go in the other PR (#16 ). Does that sound good?

**Comment by rfellows** [comment]
I've update this PR with the proposed changes.

File: src/platform/core/common/styles/_td-chips.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    width: calc(100% - 42px);   // account for padding and border as to line up with the underline
+    padding-right: 30px;        // Don't allow the text to overlap the filter icon
+  }
+}
+
+body[fds] .td-chip span {
+  font-size: 10px;
+}
+
+body [fds] mat-form-field.td-chips-form-field.mat-input-container {
```
Comment (scottyaslan) [inline src/platform/core/common/styles/_td-chips.scss]: Please delete this whole rule... you had a space in between body and [fds] so the rule was not applying correctly which caused me to look at this and determine we just want to remove the width for all inputs... I placed a comment on your input PR as well...

**Pull request closed without merging**